### PR TITLE
Link to workflow patterns in EDA

### PR DESCRIPTION
**Pull request by timbussmann**
https://learn.microsoft.com/en-us/azure/architecture/guide/architecture-styles/event-driven

### CODE REVIEW

File: docs/guide/architecture-styles/event-driven-content.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ In the logical diagram above, each type of consumer is shown as a single box. In
 
 - Guaranteed delivery. In some systems, especially in IoT scenarios, it's crucial to guarantee that events are delivered.
 - Processing events in order or exactly once. Each consumer type typically runs in multiple instances, for resiliency and scalability. This can create a challenge if the events must be processed in order (within a consumer type), or if the processing logic isn't idempotent.
+- Business processes in event-driven architectures consist of many events within the system. [Workflow patterns](https://docs.particular.net/architecture/workflows/) help to reliably coordinate message flows across various services.
```
Comment (timbussmann) [inline docs/guide/architecture-styles/event-driven-content.md:44]:
```suggestion
- Multiple events related to a single business process. [Workflow patterns](https://docs.particular.net/architecture/workflows/) can be used to reliably coordinate message flows across various services.
```

**Comment by timbussmann** [comment]
raised from personal account.

**Comment by udidahan** [comment]
> raised from personal account.

Could that be linked here as well?

**Comment by timbussmann** [comment]
@udidahan I've added links to the PR.

**Pull request closed without merging**